.release-please-manifest.json

```diff
@@ -1,16 +1,16 @@
 {
-  "packages/api-explorer": "21.0.10",
-  "packages/extension-api-explorer": "21.0.10",
-  "packages/extension-sdk": "21.0.10",
-  "packages/extension-sdk-react": "21.0.10",
-  "packages/hackathon": "21.0.10",
-  "packages/run-it": "21.0.10",
-  "packages/sdk": "21.0.10",
-  "packages/sdk-codegen": "21.0.10",
-  "packages/sdk-codegen-scripts": "21.0.10",
+  "packages/api-explorer": "21.1.0",
+  "packages/extension-api-explorer": "21.4.0",
+  "packages/extension-sdk": "21.4.0",
+  "packages/extension-sdk-react": "21.4.0",
+  "packages/hackathon": "21.4.0",
+  "packages/run-it": "21.1.0",
+  "packages/sdk": "21.4.0",
+  "packages/sdk-codegen": "21.0.11",
+  "packages/sdk-codegen-scripts": "21.0.11",
   "packages/sdk-codegen-utils": "21.0.10",
-  "packages/sdk-node": "21.0.10",
+  "packages/sdk-node": "21.4.0",
   "packages/sdk-rtl": "21.0.10",
-  "packages/wholly-sheet": "21.0.10",
-  "python": "21.0.0"
+  "packages/wholly-sheet": "21.1.0",
```

Comment:
The following packages have never been published to npm so we can keep the name and just set the version however we want (again, someone tell me and I'll update it):

- `@looker/api-explorer`
- `@looker/run-it`
- `@looker/wholly-sheet`

Reply:
https://github.com/looker-open-source/sdk-codegen/pull/536